### DISCUSSION TITLE
documentation in a single PDF document?

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#373] [Q&A] [answered]
@pgundlach,

the documentation is available as a ePub document.

I wonder whether it would make sense to provide it as a single PDF document.

It would be a good example with source code (for others to tinker with).

Just in case it may make sense.

**Comment by pgundlach** [comment, marked as answer]
This has been on my todo list for a long time... It should be more doable now that the speedata Publisher can process HTML fragments. Maybe I should go and try to write a layout for docbook...

**Reply by pr-apes** [reply]
That would be great and an excellent start for another layouts (such as the one for Markdown).
